src/main/java/ru/practicum/shareit/item/dao/ItemRepositoryImpl.java

```diff
@@ -0,0 +1,26 @@
+package ru.practicum.shareit.item.dao;
+
+import org.springframework.stereotype.Component;
+import ru.practicum.shareit.item.model.Item;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+@Component
+public class ItemRepositoryImpl implements ItemRepository {
+    private HashMap<Integer, Item> items = new HashMap();
```

Comment:
Лучше обозначать интерфейс, а уже инициализировать объект с определенной реализацией. Так программа становится более гибкой и не сломается, если нужно будет подменить реализацию на другую
```suggestion
    private Map<Integer, Item> items = new HashMap();
```